backend/src/main/java/com/github/valfink/backend/fooditem/FoodItemService.java

```diff
@@ -0,0 +1,16 @@
+package com.github.valfink.backend.fooditem;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class FoodItemService {
+    private final FoodItemRepository foodItemRepository;
+
+    public List<FoodItem> getAll() {
```

Comment:
getAll what? Too generic, choose a more specific name :)

Reply:
I did think about this before naming it like that. My idea was, since the method is part of the `FoodItemService` class and can only be called through it, it is unnecessary and redundant to call the method `getAllFoodItems`. Should I change the name nonetheless?